### ISSUE TITLE
Scrolling to the end of a Slide with many fragments: Cut last cell

### ISSUE THREAD

**Issue by enryH** [damianavila/RISE#400]
Hey,

I uploaded a presentation on github and made it available. However, when scrolling to the end of long slides / subpages, the end is cut in most sizes of the browser, including fullscreen. However, 
- this does not happen using RISE locally, only when transforming it to slides using nbconverter. 
- there are browser sizes

command: 
- jupyter nbconvert --to slides index.ipynb  --reveal-prefix=reveal.js  --SlidesExporter.reveal_scroll=**True**  --SlidesExporter.reveal_transition=none 

- does not help to include:
    - --SlidesExporter.width=1024 --SlidesExporter.height=768

A local react.js is in the github folder. 

Does anyone know how the fix it?

##### fullscreen 
![scrolling_cut_at_end_2018-07-12](https://user-images.githubusercontent.com/2833836/42643660-fd979ea8-85f9-11e8-9081-4dec9264864c.png)

##### resized window to roughly square size
![screenshot from 2018-07-12 17-45-42](https://user-images.githubusercontent.com/2833836/42644367-bb3a919e-85fb-11e8-9cba-21b776b8fdb7.png)

Thanks in advance!

**Comment by parmentelat** [comment]
Hello there

I am not quite confortable with the way nbconverter and RISE interact, but since you are saying that

> this does not happen using RISE locally, only when transforming it to slides using nbconverter.

can you please elaborate why you opened an issue with RISE and not nbconverter ? 

---
I personally never use nbconverter for exporting anything; have you tried using binder for that sort of purpose ?

**Comment by damianavila** [comment]
I agree this should be opened in nbconvert but I am eager to know @enryH answer to @parmentelat's questions.
Additionally, @enryH, it would be helpful if you link the presentation when you open the issue in nbconvert, so we can research into the issue more quickly. Thanks!
